source/mite-new.js

```diff
@@ -50,6 +50,9 @@ function getProjectChoices() {
 function getServiceChoices() {
   return bluebird.promisify(mite.getServices)()
     .then(response => response.map(d => d.service))
+    .then(services => config.get().whitelistedServices != null 
```

Comment:
I don’t understand why it’s useful to filter the services here? Could you explain why that is required?